api/types/container/host_config.go

```diff
@@ -401,6 +401,12 @@ type HostConfig struct {
 	// Mounts specs used by the container
 	Mounts []mount.Mount `json:",omitempty"`
 
+	// MaskedPaths is the list of paths to be masked inside the container (this overrides the default set of paths)
+	MaskedPaths []string
+
+	// ReadonlyPaths is the list of paths to be set as read-only inside the container (this overrides the default set of paths)
+	ReadonlyPaths []string
```

Comment:
Before we merge, these should be added to the swagger.yml as well (https://github.com/moby/moby/blob/master/api/swagger.yaml)

Reply:
will do!